dpe/src/commands/certify_key.rs

```diff
@@ -263,11 +263,15 @@ pub struct CertifyKeyMldsaExternalMu87Cmd {
 mod tests {
     use super::*;
     #[cfg(feature = "dpe_profile_p256_sha256")]
-    use crate::commands::CertifyKeyP256Cmd as CertifyKeyCmd;
+    use crate::commands::{
```

Comment:
Can you see if this will compile with `cargo check --features ml-dsa`?

I suspect it will complain